src/libsystemd-network/sd-lldp.c

```diff
@@ -183,8 +179,7 @@ int lldp_handle_packet(tlv_packet *tlv, uint16_t length) {
 
                 if (type == LLDP_TYPE_END) {
                         if (len != 0) {
-                                log_lldp("TLV type end is not length 0. Length:%d received . Dropping ...",
-                                         len);
+                                log_lldp("TLV type end is not length 0. Length: '%d' received. Dropping .", len);
```

Comment:
No whitespace before a full stop. This also applies to many other locations below.